itou/templates/companies/includes/_company_info.html

```diff
@@ -26,17 +26,19 @@ <h3>{{ company.display_name }}</h3>
                 {% include "companies/includes/_company_details.html" with company=company only %}
             {% endif %}
         </ul>
-        {% if job_app_to_transfer|default:False %}
-            <a href="{% url 'apply:job_application_external_transfer_step_1_company_card' job_application_id=job_app_to_transfer company_pk=company.pk %}{% if back_url|default:"" %}?back_url={{ back_url|urlencode }}{% endif %}"
-               class="btn btn-secondary btn-block mt-4">Voir la fiche de l'entreprise</a>
-        {% else %}
-            {% url 'companies_views:card' siae_id=company.pk as company_url %}
-            {% if back_url|default:'' and not open_in_tab|default:False %}
-                {% url_add_query company_url job_seeker=job_seeker.public_id|default:'' back_url=back_url as company_url_params %}
+        {% if not hide_cta|default:False %}
```

Comment:
Sauf si ça impacte la lecture ou la compréhension je trouve mieux de raisonner en affirmation plutôt qu'en négation (et pis flemme de réfléchir aussi :grin:)
```suggestion
        {% if show_cta|default:True %}
```

Reply:
Il faut utiliser un `default_if_none` car sinon, quand on met explicitement `show_cta=False`, le filtre `default` s'enclenche et met `True` :sweat_smile:

Reply:
Merci pour l'explication Xavier